### DISCUSSION TITLE
Deleting Waypoints on Approaches

### DISCUSSION THREAD

**Discussion by JEP795** [flybywiresim/aircraft#2386] [Help] [answered]
Hi I noticed yesterday I couldn’t delete specific waypoints on final approaches. Seems like MSFS adds some extra twists and turns that aren’t in the real life approaches and I tried removing them by selecting the CLR button and the name of the waypoint in order to leave the ones I actually needed but nothing would happen. Is this a known issue?

**Comment by NathanInnes** [comment, marked as answer]
Yes this is a known issue that was reported in #1388 but from my understanding I think you’re not able to delete waypoints from a STAR anyways
